nature.md

```diff
@@ -1045,7 +1049,7 @@ Third party scripts (e.g. advertising, customer services widgets) may inject con
 <td>
 The visual presentation of User Interface Components in this product have a contrast ratio of at least 3:1 against adjacent color(s).
```

Comment:
```suggestion
The visual presentation of User Interface Components in this product have a contrast ratio of at least 3:1 against adjacent colours.
```
We seem to be using British English for our copy this document, and the parenthesis feels... wrong.